mmcv/ops/merge_cells.py

```diff
@@ -95,8 +95,16 @@ def _resize(self, x, size):
         elif x.shape[-2:] < size:
             return F.interpolate(x, size=size, mode=self.upsample_mode)
         else:
-            assert x.shape[-2] % size[-2] == 0 and x.shape[-1] % size[-1] == 0
-            kernel_size = x.shape[-1] // size[-1]
+            if x.shape[-2] % size[-2] != 0 or x.shape[-1] % size[-1] != 0:
+                H_pad = (x.shape[-2] // size[-2] + 1) * size[-2] - x.shape[-2]
+                W_pad = (x.shape[-1] // size[-1] + 1) * size[-1] - x.shape[-1]
+                padding_left, padding_right = W_pad // 2, W_pad - W_pad // 2
+                padding_top, padding_bottom = H_pad // 2, H_pad - H_pad // 2
+                x = nn.ConstantPad2d(
+                    (padding_left, padding_right, padding_top, padding_bottom),
+                    0)(
+                        x)
```

Comment:
Could we use `F.pad` instead?

Reply:
Thanks for your suggestion, I will use the func from F replacing the one from nn.